emails/file-expiry.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+    Hr,
+    Link,
+    Section,
+    Text
+} from "@react-email/components";
+import { z } from "zod";
+import { Email } from "./_common/watcloud-email";
+
+const EmailProps = z.object({
+    name: z.string(),
+    paths: z.array(z.string()),
+    daysForExpiry: z.string(),
+    deletionDate: z.string()
+});
+
+type EmailProps = z.infer<typeof EmailProps>;
+
+export const FileExpiryEmail = (props: EmailProps) => {
```

Comment:
This can be called email, because we don't use it externally anyway.

```suggestion
const Email = (props: EmailProps) => {
```